<!DOCTYPE html>
<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">

<head th:replace="headerLinks :: headerLinks"></head>

<body data-bs-spy="scroll" data-bs-target=".navbar" data-bs-offset="70">
<div th:replace="navigation :: navigation "></div>


<section id="team">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <div class="intro">
                    <h6>Personality</h6>
                    <h1>IMPROVING PERSONALITY THROUGH ANCESTRY</h1>
                    <h3>The need is now – we are at the atrium:</h3>
                    <div class="subIntro">
                        <p class="mx-auto myTextSize">
                            BYU study shows family history can improve psychological well-being of young adults. Participating in family history research reduces students’ anxiety by 20%,
                            increases self-esteem by 8% (Sydney Walker et al May 10, 2023)
                            Approximately 80% of all genome-wide association studies (GWAS) have been performed with individuals of European (EUR) ancestry,
                            who account for a minority of the world’s population1. Linkage disequilibrium (LD) between variants confounds inferences about causal variants,
                            and the ancestral specificity of LD complicates the trans-ancestry portability of GWAS findings. GWAS have the potential to revolutionize the clinical utility of
                            genetic data to the individual, exemplified by current polygenic risk score (PRS) models.
                            Amariuta T, Ishigaki K, Sugishita H, Ohta T, Koido M, Dey KK, Matsuda K, Murakami Y, Price AL, Kawakami E, Terao C, Raychaudhuri S. Improving the
                            trans-ancestry portability of polygenic risk scores by prioritizing variants in predicted cell-type-specific
                            regulatory elements. Nat Genet. 2020 Dec;52(12):1346-1354. doi: 10.1038/s41588-020-00740-8. Epub 2020 Nov 30. PMID: 33257898; PMCID: PMC8049522.

                        </p>

                        <p>
                            Data are available at: IMPACT Github Repository: <a th:href="@{/https://github.com/immunogenomics/IMPACT}">https://github.com/immunogenomics/IMPACT</a>

                            HOMER: http://homer.ucsd.edu/homer/motif/; S-LDSC: https://github.com/bulik/ldsc; 1000 Genomes: http://www.internationalgenome.org/; cell-type-specifically expressed gene set annotations and LD scores: https://data.broadinstitute.org/alkesgroup/LDSCORE/LDSC_SEG_ldscores/; cell-type-specific histone modification ChIP–seq datasets: https://data.broadinstitute.org/alkesgroup/LDSCORE/; Plink: https://www.cog-genomics.org/plink2; Riken website: http://jenger.riken.jp/en/; Price Lab GWAS summary statistics: https://data.broadinstitute.org/alkesgroup/sumstats_formatted/; Neale Lab GWAS summary statistics: http://www.nealelab.is/uk-biobank; GWAS catalog: https://www.ebi.ac.uk/gwas/; Deep Learning: https://data.broadinstitute.org/alkesgroup/LDSCORE/DeepLearning/.
                            Previous studies have shown concordance of polygenic effects between EUR and EAS individuals in RA and between EUR and African American individuals in PrCa

                            Kichaev G & Pasaniuc, B. Leveraging functional-annotation data in trans-ethnic fine-mapping studies. Am. J. Hum. Genet97, 260–271 (2015). [PMC free article] [PubMed] [Google Scholar]
                            Gusev A et al. Atlas of prostate cancer heritability in European and African-American men pinpoints tissue-specific regulation. Nat. Commun 7, 10979 (2016).

                            Noah’s Ark Immunology, has been established to bridge the gap between imagination and action to make known the inner make up of who we really are.

                            Register online at the link below and schedule your appointment.

                        </p>
                    </div>

                </div>
            </div>
        </div>
    </div>
    </div>
</section>


<div th:replace="footer :: footer "></div>
</body>
</html>